index.bs

```diff
@@ -123,22 +125,16 @@ Issue: We should consider having further normative restrictions on file names th
 never be allowed using this API, rather than leaving it entirely up to underlying file
 systems.
 
-A <dfn export id=file>file entry</dfn> additionally consists of
-<dfn for="file entry" export>binary data</dfn> (a [=byte sequence=]), a
-<dfn for="file entry">modification timestamp</dfn> (a number representing the number of milliseconds since the <a spec=FileAPI>Unix Epoch</a>),
-a <dfn for="file entry">lock</dfn> (a string that may exclusively be "`open`", "`taken-exclusive`" or "`taken-shared`")
-and a <dfn for="file entry">shared lock count</dfn> (a number representing the number shared locks that are taken at a given point in time).
-
-A user agent has an associated <dfn>file system queue</dfn> which is the
+A user agent has an associated <dfn>file system queue</dfn>  which is the
 result of [=starting a new parallel queue=]. This queue is to be used for all
 file sytem operations.
 
 <div algorithm>
-To <dfn for="file entry/lock">take</dfn> a [=file entry/lock=] with a |value| of
-"`exclusive`" or "`shared`" on a given [=file entry=] |file|:
+To <dfn for="file system entry/lock">take</dfn> a [=file system entry/lock=] with a |value| of
```

Comment:
FWIW it would be nice to reword this definition to make lock take/release more clear (as was earlier suggested here  https://github.com/whatwg/fs/pull/95#discussion_r1106855972). For the sake of keeping the diffs clean, let's do this in a follow-up?

Reply:
... though if we're adding a new method, maybe we should just fix things in this PR rather than add a new method that follows the wrong pattern? Sorry for the confusion :)

Reply:
Does this look good?

Reply:
Sorry for not clarifying earlier, but I believe this should be `<dfn for="file system entry">take a lock</dfn>`

Spec language maps roughly to object-oriented programming, in that `[=file system entry/lock/take=]` suggests `FileSystemLock.Take()`, while `[=file system entry/take a lock=]` suggest `FileSystemEntry.TakeALock()`. The latter is more appropriate here since we're passing in a "file system entry" to this algorithm, not a lock. That confusion is what lead to https://github.com/whatwg/fs/pull/95#discussion_r1106855972 (assuming I'm reading that correctly)